client/build.gradle

```diff
@@ -14,11 +14,10 @@ plugins {
 }
 
 dependencies {
-    implementation project(path: ":${rootProject.name}-spi", configuration: 'shadow')
-    compileOnly project(':opensearch-ml-common')
+    implementation project(':opensearch-ml-common')
     compileOnly group: 'org.opensearch', name: 'opensearch', version: "${opensearch_version}"
     testImplementation group: 'junit', name: 'junit', version: '4.13.2'
-    testImplementation group: 'org.mockito', name: 'mockito-core', version: '4.4.0'
+    testImplementation group: 'org.mockito', name: 'mockito-core', version: '5.3.1'
```

Comment:
Just ask, why improve to 5.3.1, see any issue?

Reply:
This was carried over from main actually where we have mockito set to 5.3.1. Its not a must a though so I reversed this for now